apps/backoffice-v2/src/main.tsx

```diff
@@ -16,6 +16,13 @@ import './index.css';
 import dayjs from 'dayjs';
 import advancedFormat from 'dayjs/plugin/advancedFormat';
 import posthog from 'posthog-js';
+import * as Sentry from '@sentry/react';
+import {
+  createRoutesFromChildren,
+  matchRoutes,
+  useLocation,
+  useNavigationType,
+} from 'react-router-dom';
 
 if (
```

Comment:
It will be nice to extract it to a single function under the local env check, something like "initializeOptionalMonitoring"